libraries/AP_HAL_SITL/SITL_cmdline.cpp

```diff
@@ -112,7 +112,9 @@ static const struct {
     { "djix",               MultiCopter::create },
     { "cwx",                MultiCopter::create },
     { "hexa",               MultiCopter::create },
+    { "djihexa",            MultiCopter::create },
```

Comment:
Looking pretty good.  I'd prefer if we kept the frame name at the front and then add the "dji" part.  so let's make it "hexa-dji" and "octa-dji".  I see that some of the other existing frame names like "bfxrev" and "djix" are missing the quad so we should fix that as well but that doesn't need to be part of this PR, in fact, it's probably best that it's not.

Reply:
If you can update this then I'll merge it, thanks!!